app/model/schema/position.py

```diff
@@ -47,17 +50,45 @@ class SecurityTokenPosition(BaseModel):
     )
 
 
-T = TypeVar("T")
+SecurityTokenResponseT = TypeVar(
+    "SecurityTokenResponseT",
+    RetrieveStraightBondTokenResponse,
+    RetrieveShareTokenResponse,
+)
```

Comment:
- Renamed from **T** to **SecurityTokenResponseT**. https://github.com/BoostryJP/ibet-Wallet-API/pull/1380/commits/ff358781881e3da2b326508285dcb18aa838c73e